CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+This repository is a meta repository to bring together the different metadata
+schema collections and maintaining the python code-base.
+
+If you want to contribute to the schemas themselves please use the repository
+of the corresponding collection:
+
+- [**openMINDS_core**](https://github.com/HumanBrainProject/openMINDS_core)
+- [**openMINDS_SANDS**](https://github.com/HumanBrainProject/openMINDS_SANDS)
+- [**openMINDS_controlledTerms**](https://github.com/HumanBrainProject/openMINDS_controlledTerms)
+
+If you want to contribute to this repository, feel free to fork it and open a
+pull request with your changes.
+You are also welcome to contribute to discussion in the issue tracker and of
+course to report issues you are facing yourself.
+
+Another important contribution is to spread the word about this project.
+
+If you find our project helpful you can reference it in articles,
```

Comment:
I would delete this sentence

Reply:
Done :+1:

Reply:
@lzehl did mean the sentence in line 16?